NeoDebuggerUI/ViewModels/ViewModelBase.cs

```diff
@@ -9,6 +10,13 @@ namespace NeoDebuggerUI.ViewModels
 	public class ViewModelBase : ReactiveObject
 	{
         public async void OpenGenericSampleDialog(String text, String okText, String cancelText, bool showCancel)
+        {
+            var clickedOk = await GetGenericSampleDialogResult(text, okText, cancelText, showCancel);
```

Comment:
This method name obfuscate it´s real purpose that is to build the dialog, not to just get it´s result.